quick-start.md

```diff
@@ -58,13 +58,42 @@ Send the policy to Vault: `vault policy-write kubernetes-vault policy.hcl`
 Create a token role for Kubernetes-Vault that generates a 6 hour periodic token: `vault write auth/token/roles/kubernetes-vault allowed_policies=kubernetes-vault period=6h`
 
 ### 2.6 Generate the token for Kubernetes-Vault and AppID
-Generate the token: `vault token-create -role=kubernetes-vault`
+Generate the token: `vault token-create -role=kubernetes-vault`. And make a note of the token output. In the example below it would be '00000000-1111-2222-3333-444444444444'
+
+```
+$ vault token-create -role=kubernetes-vault
+
+Handling connection for 8200
+Key             Value
+---             -----
+token           00000000-1111-2222-3333-444444444444      
+token_accessor  c50fb600-aaaa-bbbb-cccc-xxxxxxxxxxxx
+token_duration  6h0m0s
+token_renewable true
+token_policies  [default kubernetes-vault]
+```
 
 Get the AppID: `vault read auth/approle/role/sample-app/role-id`
 
```

Comment:
Can you add the same example for getting the AppID and adding it to the `sample-app`'s deployment as well?